java/ql/test/query-tests/security/CWE-611/options

```diff
@@ -1 +1 @@
-//semmle-extractor-options: --javac-args -cp ${testdir}/../../../stubs/jdom-1.1.3:${testdir}/../../../stubs/dom4j-2.1.1:${testdir}/../../../stubs/simple-xml-2.7.1:${testdir}/../../../stubs/jaxb-api-2.3.1
+//semmle-extractor-options: --javac-args -cp ${testdir}/../../../stubs/jdom-1.1.3:${testdir}/../../../stubs/dom4j-2.1.1:${testdir}/../../../stubs/simple-xml-2.7.1:${testdir}/../../../stubs/jaxb-api-2.3.1:${testdir}/../../../stubs/jaxen-1.2.0
```

Comment:
This looks like the only change in the CWE-611 directory. Was this committed by mistake?

Reply:
I changed this because tests for CWE-611 started failing when I introduced a Jaxen import in a stub used both in CWE-611 and CWE-643 (here https://github.com/github/codeql/pull/5774/commits/76468559ba8ba993bb9fae0b78d3f3482eb0ec48#diff-da791bf0774d1d6fe164bf5fa3cae3cec838ee5f8a7218737364701624c5716bR95)